Minesweeper.java

```diff
@@ -119,9 +159,11 @@ private Menu()
 
             JPanel difficultyPanel = new JPanel();
 
-            JSlider difficultySlider = new JSlider(JSlider.HORIZONTAL, 5, 35, gridSize);
+            JSlider gridSizeSlider = new JSlider(JSlider.HORIZONTAL, 5, 35, this.selectedGridSize);
+            JSlider bombAmountSlider = new JSlider(JSlider.HORIZONTAL, 0, 99, this.selectedBombAmount);
```

Comment:
Do we want the user to select the amount of bombs manually? Shouldn't this be linked to the difficulty and be abstracted from the user?

Reply:
im not sure what you mean by "abstracted from the user". The amount of bombs is automatically generated if you choose one of the preset difficulty levels, whereas if the user selects a custom difficulty they can choose a custom grid size as well as a custom amount of bombs to go with it. I think it's more complete this way; allows the user full control when choosing a custom difficulty.

Reply:
ah yeah mb, i thought the slider was accesible without making the difficulty custom